packages/mobile/ios/Utils.swift

```diff
@@ -0,0 +1,17 @@
+@objc(Utils)
+class Utils: NSObject {
+
+    @objc
+    func generateRandomString(length: Int) -> String {
+    let letters = "0123456789"
+    var randomString = String()
+
+    for _ in 0..<length {
+        let rand = Int(arc4random_uniform(UInt32(letters.count)))
+        let character = letters[letters.index(letters.startIndex, offsetBy: rand)]
+        randomString.append(character)
+    }
+
+    return randomString
+}
+}
```

Comment:
Same comment as above

> We should probably use something like: https://developer.apple.com/documentation/security/1399291-secrandomcopybytes
>
> and maybe call it `generateSecret` to match other implementations.

Also it looks like this is a duplicate of the same function in AppDelegate.m